### PR TITLE
Describe the DataGrid - export to pdf demo

### PR DESCRIPTION
**Pull request by AlexJameson**

### CODE REVIEW

File: JSDemos/Demos/DataGrid/ExportToPDF/description.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Our [DataGrid](/Documentation/ApiReference/UI_Widgets/dxDataGrid/) allows you to easily and accurately export its contents to a PDF file. To enable export operations, you must reference or import <a href="https://github.com/MrRio/jsPDF" target="_blank">jsPDF</a> library.
+
+Once you’ve referenced/imported `jsPDF`, call the **DevExpress**.**pdfExporter**.**exportDataGrid(options)** method to export **DataGrid** content in the PDF format.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/DataGrid/ExportToPDF/description.md]:
```suggestion
Once you've referenced/imported `jsPDF`, call the **DevExpress**.**pdfExporter**.**exportDataGrid(options)** method to export **DataGrid** content in the PDF format.
```

File: JSDemos/Demos/DataGrid/ExportToPDF/description.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+Review the export code in the button's [onClick](/Documentation/ApiReference/UI_Widgets/dxButton/Configuration/#onClick) handler to learn more.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/DataGrid/ExportToPDF/description.md]:
```suggestion
In this demo, the **exportDataGrid(options)** method is called when you click the Export to PDF button. Review the export code in the button's [onClick](/Documentation/ApiReference/UI_Widgets/dxButton/Configuration/#onClick) handler to learn more.
```

File: JSDemos/Demos/DataGrid/ExportToPDF/description.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Our [DataGrid](/Documentation/ApiReference/UI_Widgets/dxDataGrid/) allows you to easily and accurately export its contents to a PDF file. To enable the export operation, you must reference or import <a href="https://github.com/MrRio/jsPDF" target="_blank">jsPDF</a> library.
```
Comment (Abadzhev) [inline JSDemos/Demos/DataGrid/ExportToPDF/description.md]: 'easily and accurately' - taken from Ray's description? 

I think you need an article here: "the jsPDF library". It differs when the usage below, where you don't add the word "library".

Comment (AlexJameson) [reply JSDemos/Demos/DataGrid/ExportToPDF/description.md]: @Abadzhev Hello again, I attached the screenshot with a similar description rewritten by Ray.

image

Comment (Abadzhev) [reply JSDemos/Demos/DataGrid/ExportToPDF/description.md]: Okay - not going to argue with that :-)

File: JSDemos/Demos/DataGrid/ExportToPDF/description.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Our [DataGrid](/Documentation/ApiReference/UI_Widgets/dxDataGrid/) allows you to easily and accurately export its contents to a PDF file. To enable the export operation, you must reference or import the <a href="https://github.com/MrRio/jsPDF" target="_blank">jsPDF</a> library.
```
Comment (arminal) [inline JSDemos/Demos/DataGrid/ExportToPDF/description.md]:
```suggestion
Our [DataGrid](/Documentation/ApiReference/UI_Widgets/dxDataGrid/) allows you to easily and accurately export its contents to PDF. To enable the export operation, you must reference or import the <a href="https://github.com/MrRio/jsPDF" target="_blank">jsPDF</a> library.
```

File: JSDemos/Demos/DataGrid/ExportToPDF/description.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+
+Once you've referenced/imported `jsPDF`, call the **DevExpress**.**pdfExporter**.**exportDataGrid(options)** method to export **DataGrid** content in the PDF format.
```
Comment (arminal) [inline JSDemos/Demos/DataGrid/ExportToPDF/description.md]:
```suggestion
Once you've referenced/imported `jsPDF`, call the **DevExpress**.**pdfExporter**.**exportDataGrid(options)** method to export **DataGrid** content to PDF.
```

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**